src/jit/emitarm.cpp

```diff
@@ -7581,7 +7581,6 @@ void emitter::emitInsMov(instruction ins, emitAttr attr, GenTree* node)
             else
             {
                 assert(!data->isContained());
-                codeGen->genConsumeReg(data);
                 emitIns_S_R(ins, attr, data->gtRegNum, varNode->GetLclNum(), 0);
```

Comment:
It triggered an assertion about a node being consumed twice. I removed it by looking at the x86 version of emitInsMov.